tests/features/step_definitions/my_steps.rb

```diff
@@ -11,16 +11,18 @@
   x = rand(1..2)
 
   if x == 1
-    on(ProjectsSettingsPage).open_projects_issues_new_page
+    on(ProjectsSettingsPage).new
```

Comment:
sorry for my not perfect idea (
I don't think that ```new``` method is ok for this case
because this method name is commonly used in ruby as a constructor method
and if we use this one for any other purpose we should consider other method name as well